package.json

```diff
@@ -55,6 +55,7 @@
   },
   "peerDependencies": {
     "codemirror": "^5.39.2",
+    "d3": "^3.5.17",

```

Comment:
remove ^ ?

Reply:
For the peer dependencies I think it make sense to allow more versions